Library/Formula/showkey.rb

```diff
@@ -0,0 +1,62 @@
+class Showkey < Formula
+  desc "Command-line keystroke visualizer"
+  homepage "http://www.catb.org/~esr/showkey"
+  url "http://www.catb.org/~esr/showkey/showkey-1.7.tar.gz"
+  sha256 "0383a508c29df9a80b294a805a38f38d2dee6d2687e43c5ece3c5901220fb44d"
+
+  head do
+    url "git://thyrsus.com/repositories/showkey.git"
+    depends_on "xmlto" => :build
+    # Disable checking XML validity by xmlto, as xslproc is often configured to
+    # avoid downloading DTDs. This can result in a xml to man conversion error.
+    patch :DATA
```

Comment:
It doesn't seem to be a showkey compilation issue per se. I couldn't locate whether it is an issue with xmlto or xslproc in general. It can be about the switches with which these two formulas are compiled in homebrew (xslt had a switch called 'nonet'). It seems almost all complaints online have ended with disabling validation.